Plugins.md

```diff
@@ -20,14 +20,14 @@ To set up plugin to create gist under your user name you need to give it your gi
 * Create [Github personal access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) and **make sure token has _gist_ scope selected**. Authenticating with username and password isn't supported for security reasons. Set it as `$Env:github_api_key` environment variable.
 
 
-## [Git](https://github.com/majkinetor/au/blob/master/AU/Plugins/Git.ps1)
+## [Git](src/Plugins/Git.ps1)
 
 **Persist modified files**.
 
-* To use it locally, just ensure `git push` doesn't require credentials and dont set any environment variables. 
+* To use it locally, just ensure `git push` doesn't require credentials and dont set any environment variables.
```

Comment:
```suggestion
* To use it locally, just ensure `git push` doesn't require credentials and don't set any environment variables.
```

Although, @pauby maybe want it to be `do not` 🤷

Reply:
I've updated it to `don't`. I would suggest we leave it as `don't` as it was that way in the original repository, and this PR was mostly about adding a Migration document and updating a few references from `AU` to `Chocolatey-AU`, but will update it to `do not` if that is preferred.